consensus/src/round_manager.rs

```diff
@@ -1257,6 +1257,15 @@ impl RoundManager {
                         qc
                     ))?;
                 if self.onchain_config.order_vote_enabled() {
+                    // This check is already done in safety rules. As printing the "failed to broadcast order vote"
+                    // in humio logs could sometimes look scary, we are doing the same check again here.
+                    if let Some(last_sent_vote) = self.round_state.vote_sent() {
+                        if let Some((two_chain_timeout, _)) = last_sent_vote.two_chain_timeout() {
```

Comment:
nit: this can be more compact with 
```
if self.round_state.vote_send()
    .map(|last_vote| last_vote.map(|(timeout, _)|) round <= timeout.round()).flatten().unwrap_or(false) {
}
```